src/components/deploymentConfig/DeploymentTemplateView/DeploymentTemplateOptionsTab.tsx

```diff
@@ -49,38 +51,37 @@ export default function DeploymentTemplateOptionsTab({
 
     const restoreLastSaved = () => {
         if (isEnvOverride) {
-            const overriddenValues = !!state.latestDraft
+            const overriddenValues = state.latestDraft
                 ? state.draftValues
                 : YAML.stringify(state.duplicate, { indent: 2 })
             const _envValues =
                 state.data.IsOverride || state.duplicate
                     ? overriddenValues
                     : YAML.stringify(state.data.globalConfig, { indent: 2 })
-            dispatch({
-                type: DeploymentConfigStateActionTypes.tempFormData,
-                payload: _envValues,
-            })
-        } else {
-            dispatch({
-                type: DeploymentConfigStateActionTypes.tempFormData,
-                payload: !!state.latestDraft ? state.draftValues : YAML.stringify(state.template, { indent: 2 }),
-            })
+            if (isValues){     
```

Comment:
we can add this check at the first line return if !isValues

Reply:
done